code/modules/surgery/organs/external/wings/moth_wings.dm

```diff
@@ -35,14 +35,24 @@
 /obj/item/organ/external/wings/moth/proc/update_float_move()
 	SIGNAL_HANDLER
 
-	if(!isspaceturf(owner.loc) && !burnt)
+	if(can_fly())
 		var/datum/gas_mixture/current = owner.loc.return_air()
 		if(current && (current.return_pressure() >= ONE_ATMOSPHERE*0.85)) //as long as there's reasonable pressure and no gravity, flight is possible
 			ADD_TRAIT(owner, TRAIT_FREE_FLOAT_MOVEMENT, REF(src))
 			return
 
 	REMOVE_TRAIT(owner, TRAIT_FREE_FLOAT_MOVEMENT, REF(src))
 
+///Checks if our wings are usable
```

Comment:
undocumented changes

Reply:
Its a sanity check, otherwise the wings still work even while hidden.

Reply:
It's a change to existing items that already hide your wings, so it needs to be in the changelog

Reply:
yeah mb